tensorflow/core/common_runtime/dml/dml_device_state.cc

```diff
@@ -70,8 +70,14 @@ namespace tensorflow {
   ComPtr<IDMLDevice> dml_device;
   dml_device = CreateDmlDevice(d3d_device.Get(), dml_flags);
 
+  D3D12_COMMAND_LIST_TYPE queue_type = D3D12_COMMAND_LIST_TYPE_DIRECT;
+
+  if (adapter.VendorID() == VendorID::kAmd) {
+    queue_type = D3D12_COMMAND_LIST_TYPE_COMPUTE;
```

Comment:
Let's keep an environment variable to force the queue on/off as well, but default to on for AMD. Might be useful for experimentation.